Client/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp

```diff
@@ -4922,15 +4922,22 @@ bool CStaticFunctionDefinitions::FadeCamera(bool bFadeIn, float fFadeTime, unsig
     return true;
 }
 
-bool CStaticFunctionDefinitions::SetCameraViewMode(unsigned short ucMode)
+bool CStaticFunctionDefinitions::SetCameraVehicleViewMode(eVehicleCamMode ucMode)
 {
-    m_pCamera->SetCameraViewMode((eVehicleCamMode)ucMode);
+    m_pCamera->SetCameraVehicleViewMode(ucMode);
     return true;
 }
 
-bool CStaticFunctionDefinitions::GetCameraViewMode(unsigned short& ucMode)
+bool CStaticFunctionDefinitions::SetCameraPedViewMode(ePedCamMode ucMode)
 {
-    ucMode = m_pCamera->GetCameraViewMode();
+    m_pCamera->SetCameraPedViewMode(ucMode);
+    return true;
+}
+
+bool CStaticFunctionDefinitions::GetCameraViewMode(unsigned short& ucVehicleMode, unsigned short& ucPedMode)
+{
+    ucVehicleMode = (unsigned short)m_pCamera->GetCameraVehicleViewMode();
+    ucPedMode     = (unsigned short)m_pCamera->GetCameraPedViewMode();
     return true;
 }
```

Comment:
Unless those methods are being used in more than one place, you can just get rid of the methods in CStaticFunctionDefinitions and just move the content to the actual luadef.